azure-pipelines/config/tsaoptions.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
nit: uppercase the "O" in "tsaOptions.json" for the file name

Reply:
oh but ignore this comment if that's the way it's supposed to be for the config to get picked up

Reply:
Yeah this is the configuration used in ADS too. I will keep it as-is for now, and if it works as expected, will update the name later :-).